src/components/navigation/header/menu/item.tsx

```diff
@@ -0,0 +1,89 @@
+import { faCaretDown } from '@fortawesome/free-solid-svg-icons'
+import * as NavigationMenu from '@radix-ui/react-navigation-menu'
+import clsx from 'clsx'
+import { default as NextLink } from 'next/link'
+import { PointerEventHandler } from 'react'
+
+import { Icon } from './icon'
+import { SubContent } from './sub-content'
+import { FaIcon } from '@/components/fa-icon'
+import { HeaderLinkData } from '@/data-types'
+import { submitEvent } from '@/helper/submit-event'
+
+export const styledLinkCls = /* className={ */ clsx(
+  'navtrigger flex align-middle items-center',
+  'w-full text-[1.33rem] font-bold',
+  'text-brand block border-b border-brand-400 p-4',
+  'md:serlo-menu-entry-special',
+  'md:block md:serlo-button-blue-transparent md:text-[0.9rem] md:leading-tight md:transition md:text-brand-500',
+  'md:my-0 md:mt-[2px] md:py-0.5 md:px-[7px]',
+  'md:text-center',
+  'hover:no-underline hover:bg-brand-300'
+)
+
+export const preventHover: PointerEventHandler = (event) => {
+  if (window.innerWidth < 1024) event.preventDefault()
+}
+
+export interface ItemProps {
+  link: HeaderLinkData
+  specialContent?: JSX.Element
+}
+
+export function Item({ link, specialContent }: ItemProps) {
+  const hasChildren = link.children !== undefined
+
+  const textAndIcon = (
+    <>
+      <Icon icon={link.icon} element={specialContent} alt={link.title} />
+      <span className={clsx(specialContent && 'md:sr-only')}>{link.title}</span>
```

Comment:
```suggestion
      <span className={clsx(specialContent ?? 'md:sr-only')}>{link.title}</span>
```

Reply:
See https://mariusschulz.com/blog/nullish-coalescing-the-operator-in-typescript

Reply:
What the current version wants to archive: Add `md:sr-only` if `specialContent` is defined. 
The safest way would be `specialContent ? 'md:sr-only' : ''` which we actually use in JSX now to avoid accidentally rendering the left hand side. But I think inside `clsx` this  shorthands is safe.